fe/fe-core/src/main/java/org/apache/doris/catalog/Replica.java

```diff
@@ -74,7 +74,7 @@ public boolean canQuery() {
     @SerializedName(value = "version")
     private volatile long version;
     @SerializedName(value = "versionHash")
-    private long versionHash;
```

Comment:
We can just remove it.

Reply:
Could not remove it because we use read write method to serialize the metadata.